taxonomies/ingredients.txt

```diff
@@ -28019,6 +28021,10 @@ hu:főtt fehér rizs
 nl:gekookte witte rijst
 # ingredient/cooked-white-rice has 75 products in 2 languages @2018-12-19
 
+<en:cooked and seasoned rice
+en:cooked and seasoned rice
+de:gekochter gewürzter Reis, gekochter gewuerzter Reis, gekochter und gewürzter Reis, gekochter und gewuerzter Reis
```

Comment:
For this one, it's better to add a new en:seasoned entry in ingredients_processing.txt (and its German translation) so that we can match things like cooked and seasoned anything, like "seasoned vegetables", without having to add all combinations as individual entries in ingredients.txt